rust/perspective-python/perspective/widget/__init__.py

```diff
@@ -201,18 +210,19 @@ def handle_message(self, widget, content, buffers):
         if content["type"] == "connect":
             client_id = content["client_id"]
             logging.debug("view {} connected", client_id)
-            self._sessions[client_id] = self.new_proxy_session(
-                lambda msg: self.send(
+
+            def send_response(msg):
+                self.send(
                     {"type": "binary_msg", "client_id": client_id}, [msg]
                 )
-            )
+            self._sessions[client_id] = self.new_proxy_session(send_response)
         elif content["type"] == "binary_msg":
             [binary_msg] = buffers
             client_id = content["client_id"]
             session = self._sessions[client_id]
-            logging.debug("view {} message {}", client_id, len(binary_msg))
             if session is not None:
-                session.handle_request(binary_msg)
+                import asyncio
+                asyncio.create_task(session.handle_request_async(binary_msg))
```

Comment:
I believe it's the case that ipykernel typically runs python code [inside an event loop](https://blog.jupyter.org/ipython-7-0-async-repl-a35ce050f7f7?gi=faf99afc522d), so this should be OK to do whether or not the underlying table is an AsyncTable or (sync) Table.

This would break in a custom Python kernel which doesn't run an asyncio event loop. I could implement a fallback in case one is not running, but I don't know a good way offhand to set up an environment to test that.